roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -5,6 +5,14 @@
     dest: /usr/local/bin/openshift-node
     mode: 0500
 
+- name: Install cni-cleanup file
+  template:
+    dest: "/usr/lib/tmpfiles.d/cleanup-cni.conf"
+    src: "cleanup-cni.j2"
+  when: not openshift_is_atomic | bool
+  notify:
+  - reload systemd units
```

Comment:
@openshift/team-release-admins do we need this anymore since we're not using systemd units to do this?